pyluos/tools/bootloader.py

```diff
@@ -563,6 +587,12 @@ def luos_options():
                               nargs='?')
     detect_parser.set_defaults(func=luos_detect)
 
+    # declare "rescue" subcommand
+    rescue_parser = subparsers.add_parser('reset',
+                                          help='tool to reset a blocked node in rescue mode')
```

Comment:
This can work with multiple nodes too write?
If yes I propose `a tool to reset one or multiple blocked nodes in rescue mode`

Reply:
yes that is why this help text should be in the plural.

Reply:
OK, I didn't get it.